stdlib/domain.mli

```diff
@@ -67,7 +67,7 @@ val at_exit : (unit -> unit) -> unit
     the function most recently added with [at_exit] is called first. An example:
 
     {[
-let temp_file_key = Domain.DLS.new_key (fun _ ->
+let temp_file_key = Thread_local_storage.new_key (fun _ ->
```

Comment:
This example needs to be modified. The temporary files are thread-local. But the example only closes the file when the domain exits. It is asymmetric.